sway-fmt-v2/src/utils/path.rs

```diff
@@ -132,3 +135,21 @@ impl Format for PathTypeSegment {
         Ok(())
     }
 }
+
+impl CommentVisitor for PathExpr {
+    fn collect_spans(&self) -> Vec<CommentSpan> {
+        // TODO: Should we look for a comment inside the path expression? if so we will need to implement CommentVisitor for PathExpr
+        // For now we are assuming there will be no comments inside the PathExpr so the following comment will be omitted
+        // root::parent/* i am a comment*/::child
+        // I am not sure if this is something the language will allow.
```

Comment:
Wow I have never seen or would have considered a comment within a path expr :smiling_face_with_tear: Does that even parse? What does Rust do?

I'd be totally fine with removing this comment and keeping the assumption that paths must be contiguous (without space or comments between separators).

Reply:
:smile: just tested and it does not parse, so I am deleting this comment and leaving this assumption in-place.